docs/_templates/logo-text.html

```diff
@@ -0,0 +1,5 @@
+<a href="{{ homepage() }}" class="text-logo">
+  <img src='{{ pathto('_static/logo.svg', 1) }}' width='100%'>
```

Comment:
[(It's documented that way)](http://www.sphinx-doc.org/en/stable/templating.html#pathto)